components/core/Application.js

```diff
@@ -64,6 +68,11 @@ export default class ApplicationPage extends React.Component {
     window.removeEventListener("drop", this._handleDrop);
   }
 
+  _handleOnlineStatus = async () => {
+    window.alert(navigator.onLine ? "online" : "offline");
```

Comment:
Nit: Any reason for `onLine` versus just `online`? seems like an interesting distinction haha.